Makefile

```diff
@@ -131,7 +136,7 @@ CFLAGS_$(objdir)/utils/demangle.op = -Wno-unused-value
 MAKEFLAGS = --no-print-directory
 
 
-all: $(objdir)/.config $(TARGETS)
+all: $(objdir)/.config $(srcdir)/check-deps/have_libelf $(TARGETS)
```

Comment:
@namhyung There is one thing I do care about is elfutils has to be compiled before uftrace linking but I tried to put the below:
```
.NOTPARALLEL: all
```
It makes the order correctly but invalidate all the parallel compilation for uftrace source code. Could you let me know if you know better way?

Reply:
Then why not making uftrace and libmcount depend on libelf?

Reply:
Hm.. The script is getting more complicated.
```make
HAVE_LIBELF := $(srcdir)/check-deps/have_libelf

$(objdir)/libmcount/libmcount.so: $(LIBMCOUNT_OBJS) $(LIBMCOUNT_UTILS_OBJS) $(LIBMCOUNT_ARCH_OBJS) $(HAVE_LIBELF)
        $(QUIET_LINK)$(CC) -shared -o $@ $^ $(LIB_LDFLAGS)
```
Adding `have_libelf` into dependency list makes it to be compiled together that makes compilation error. I may have to think about it in a different way.

Reply:
It's getting longer but I just used `filter-out` to fix it.